packages/wdio-cucumber-framework/src/cucumberEventListener.ts

```diff
@@ -153,6 +153,10 @@ export default class CucumberEventListener extends EventEmitter {
     //     }
     // }
     onTestRunStarted () {
+        if (this._gherkinDocEvents.length > 1) {
+            console.log('✖️ Multiple features will run on a single thread/connection will emit before-feature and after-feature in different way...')
```

Comment:
I can remove this, I added it for debugging and looks like I forgot to remove it.